WARNING:urllib3.connectionpool:Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPConnection object at 0x109859550>: Failed to resolve 'http' ([Errno 8] nodename nor servname provided, or not known)")': /test.txt?location=
WARNING:urllib3.connectionpool:Retrying (Retry(total=3, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPConnection object at 0x109859640>: Failed to resolve 'http' ([Errno 8] nodename nor servname provided, or not known)")': /test.txt?location=
WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPConnection object at 0x10985a9f0>: Failed to resolve 'http' ([Errno 8] nodename nor servname provided, or not known)")': /test.txt?location=
WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPConnection object at 0x10985ae10>: Failed to resolve 'http' ([Errno 8] nodename nor servname provided, or not known)")': /test.txt?location=
WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPConnection object at 0x10985afc0>: Failed to resolve 'http' ([Errno 8] nodename nor servname provided, or not known)")': /test.txt?location=
